op-nat/justfile

```diff
@@ -29,7 +29,7 @@ clean:
 # Run tests
 # test: (go_test "./...")
 test:
-    go build ./... && go test -v ./...
+    CGO_ENABLED=0 go build ./... && go test -count=1 -v ./...
```

Comment:
I'm curious: why disable the test cache?

Reply:
While developing/testing it made the output more predictable and nice.
Then that got me wondering whether or not we should trust the cache or always re-run. Because while the test may remain the same, the system may not.

Reply:
Paranoia :)
I do plan to remove this.